interactive-basemap/demo.html

```diff
@@ -19,7 +19,7 @@
         <h1>Interactive base map</h1>
         <p>Retrieve information associated with the map features</p>
     </div>
-    <p>This example demonstrates how to enable the interactions with the base map features. The interactions are enabled for the countries'
+    <p>This example demonstrates how to enable the interactions with the base map features. See  <a href="https://github.com/heremaps/maps-api-for-javascript-examples">the README</a> for instructions on how to run the example.The interactions are enabled for the countries'
```

Comment:
```suggestion
    <p>This example demonstrates how to enable the interactions with the base map features. See  <a href="https://github.com/heremaps/maps-api-for-javascript-examples#maps-api-for-javascript">the README</a> for instructions on how to run the example. The interactions are enabled for the countries'
```

What's the reason for adding this line only to this example?